### PR TITLE
Add example on how to know the executor

### PR DESCRIPTION
**Pull request by JordyGit**

### CODE REVIEW

**Review by AntonPlanckeSLC — APPROVED** [review]

File: CustomCommandExamples/Executor Input Example_1/Executor Input Example_1.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+public class Script
+{
+	/// <summary>
+	/// The Script entry point.
+	/// </summary>
+	/// <param name="engine">Link with SLAutomation process.</param>
+	public void Run(Engine engine)
+	{
+		// This input will not be asked from the user but will automatically be filled in by the DataMiner bot with the
+		// dataminer.services email address of the user executing this automation script using their linked DataMiner account
```
Comment (MariekeGO) [inline CustomCommandExamples/Executor Input Example_1/Executor Input Example_1.cs:15]: When "Automation" refers to the DataMiner Automation module, we usually capitalize it.
```suggestion
		// dataminer.services email address of the user executing this Automation script using their linked DataMiner account
```

File: CustomCommandExamples/Executor Input Example_1/Executor Input Example_1.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		// Note that this needs to be defined as input on the automation script with the exact name "dataminer.services User Email",
```
Comment (MariekeGO) [inline CustomCommandExamples/Executor Input Example_1/Executor Input Example_1.cs:16]:
```suggestion
		// Note that this needs to be defined as input on the Automation script with the exact name "dataminer.services User Email",
```

File: CustomCommandExamples/Executor Input Example_1/Executor Input Example_1.cs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		// otherwise it will not be provided by the DataMiner bot
+		var userEmail = engine.GetScriptParam("dataminer.services User Email");
+		if (string.IsNullOrWhiteSpace(userEmail?.Value))
+		{
+			engine.ExitFail("'dataminer.services User Email' parameter is required.");
+			return;
+		}
+
+		// The DataMiner user executing an automation script is always available
```
Comment (MariekeGO) [inline CustomCommandExamples/Executor Input Example_1/Executor Input Example_1.cs:25]:
```suggestion
		// The DataMiner user executing an Automation script is always available
```

File: CustomCommandExamples/README.md [hunk 4]
```diff
@@ -16,9 +16,11 @@ This readme details how to create custom DataMiner bot commands using Automation
   - [Microsoft Teams](#microsoft-teams-1)
     * [Getting Started](#getting-started)
     * [Output an Adaptive Card](#output-an-adaptive-card)
+    * [Output an Adaptive Card with buttons](#output-an-adaptive-card-with-buttons)
```
Comment (MariekeGO) [inline CustomCommandExamples/README.md:19]:
```suggestion
    * [Output an Adaptive Card with Buttons](#output-an-adaptive-card-with-buttons)
```

File: CustomCommandExamples/README.md [hunk 5]
```diff
@@ -85,6 +87,14 @@ See [Installing DataMiner Integration Studio](https://aka.dataminer.services/Dis
 
 2. The custom command will be executed and the Adaptive Card example should be displayed as the response. <details><summary>`show demo`</summary>![Gif-OutputAnAdaptiveCard](https://user-images.githubusercontent.com/33500507/192817296-cd05a0cb-2267-4639-8d21-32b54bb347cd.gif)</details>
 
+#### Output an Adaptive Card with buttons
```
Comment (MariekeGO) [inline CustomCommandExamples/README.md:90]: As title casing is used for all the other titles in this file, I'd use it here as well:
```suggestion
#### Output an Adaptive Card with Buttons
```

File: CustomCommandExamples/README.md [hunk 6]
```diff
@@ -85,6 +87,14 @@ See [Installing DataMiner Integration Studio](https://aka.dataminer.services/Dis
...
+
+1. First make sure that you fill in the organization ID and DMS ID in the Automation script named '*Adaptive Card Output Example with buttons*'. You can find these IDs in the URL by opening the DMS overview in the [Admin app](https://admin.dataminer.services). An example is provided in the Automation script.
```
Comment (MariekeGO) [inline CustomCommandExamples/README.md:92]:
```suggestion
1. Fill in the organization ID and DMS ID in the Automation script named '*Adaptive Card Output Example with buttons*'. You can find these IDs in the URL by opening the DMS overview in the [Admin app](https://admin.dataminer.services). An example is provided in the Automation script.
```

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**